app/views/queue_items/index.html.haml

```diff
@@ -4,30 +4,32 @@
       %article
         %header
           %h2 My Queue
-        %table.table
-          %thead
-            %tr
-              %th(width="10%") List Order
-              %th(width="30%") Video Title
-              %th(width="10%") Play
-              %th(width="20%") Rating
-              %th(width="15%") Genre
-              %th(width="15%") Remove
-          %tbody
-            - @queue_items.each do |item|
+        = bootstrap_form_tag update_queue_path
```

Comment:
It does, you just didn't have it defined. I added that and `f` was a `FormBuilder` instance.